File: ReactiveCocoa的简单学习/ReactiveCocoa的简单学习/ViewController.m
//
//  ViewController.m
//  ReactiveCocoa的简单学习
//
//  Created by 高李军 on 15/11/16.
//  Copyright © 2015年 高李军. All rights reserved.
//

#import "ViewController.h"
#import "NSObject+CaculatorMaker.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    int result = [NSObject caculotors:^(Caculoter *maker) {
        maker.add(3).add(4);
    }];
    
    NSLog(@"===========%d",result);
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: AlamofireLianxi/AlamofireLianxi/ViewController.swift
//
//  ViewController.swift
//  AlamofireLianxi
//
//  Created by 高李军 on 15/10/20.
//  Copyright © 2015年 LJLianXi. All rights reserved.
//

import UIKit
import Alamofire

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        Alamofire.request(.GET, "http://httpbin.org/get")
            .responseString { response in
                print("Success: \(response.result.isSuccess)")
                print("Response String: \(response.result.value)")
        }
        
        
        Alamofire.request(.GET, "http://httpbin.org/get", parameters: ["foo": "bar"])
            .responseData { response in
                print(response.request)
                print(response.response)
                print(response.result)
        }
        
        Alamofire.request(.GET, "http://httpbin.org/get")
            .responseString { response in
                print("Response String: \(response.result.value)")
            }
            .responseJSON { response in
                print("Response JSON: \(response.result.value)")
        }
        
        
    }
    
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

